### PR TITLE
EZP-32120: Security checklist v2

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-32120
| Versions      | 1.13, 2.5

Follow-up to https://github.com/ezsystems/developer-documentation/pull/1209 for v1 and v2

**Please note**: Commit https://github.com/ezsystems/developer-documentation/commit/9d47e931d54133fbb31083cf22b76ea312a0da25 is the same as in #1209, please only review commit https://github.com/ezsystems/developer-documentation/commit/29b03c5cd7e5fcee4c51a9b672cda5e7f0690423

### CODE REVIEW

**Review by glye — APPROVED** [review]
We don't have password rules and Cloud for v2? o_0

**Comment by DominikaK** [comment]
You original notes specifically marked those two tips as 3+ only

**Comment by glye** [comment]
Ah, those would be from AR. OK then.

File: docs/guide/security_checklist.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+- Configure the server to alert you about security updates from vendors.
+Pay special attention to dependencies used by your project directly, or by PHP.
+
+## eZ Publish Legacy
+
+If you are using Legacy Bridge, there are a few more things to check:
+
+- Is there a measure in place against brute force login attacks? `MaxNumberOfFailedLogin > 0` will do.
+- Do `MinPasswordLength` and `GeneratePasswordLength` (if used) have sane values? The default values are low.
+A secure minimum is 16. Do not use  value below 10.
```
Comment (dabrt) [inline docs/guide/security_checklist.md:122]:
```suggestion
16 characters is a secure minimum. Do not go below 10.
```

**Review by dabrt — APPROVED** [review]

**Pull request merged**